frontend/tutorial/step_7.md

```diff
@@ -14,17 +14,19 @@ const baseProps = {
 
 describe('Hello', () => {
 
+  const getLocals = Hello.prototype.getLocals;
```

Comment:
this adds some burden IMO,
also it is potentially error-prone: in many `getLocals` around the apps we access `this`, typically for other helper methods

It looks safer if we just suggest to `instance.getLocals(props)`?